mlir/include/mlir/Analysis/TopologicalSortUtils.h

```diff
@@ -104,6 +104,14 @@ bool computeTopologicalSorting(
     MutableArrayRef<Operation *> ops,
     function_ref<bool(Value, Operation *)> isOperandReady = nullptr);
 
+/// Get a list of blocks that is sorted according to dominance. This sort is
+/// stable.
+SetVector<Block *> getBlocksSortedByDominance(Region &region);
+
+/// Sorts all operation in `toSort` topologically while also region semantics.
+/// Does not support multi-sets.
```

Comment:
Parse error! I think this sentence has a typo somewhere :)
Also `all operationS`.

Reply:
Ah, I'm being nitpicky today: could you also make the tense consistent with the comment above? Either `Get` and `Sort` or `Gets` and `Sorts`.